packages/lab/src/list/List.tsx

```diff
@@ -223,7 +222,7 @@ export const List = forwardRef(function List<
     };
     list.push(
       isChildItem ? (
-        cloneElement(value, forwardCallbackProps(value.props, listItemProps))
+        cloneElement(value, listItemProps)
```

Comment:
forwardCallbackProps not required here as we do not include any callback props in the overrides